sections/Titans stronghold.tex

```diff
@@ -0,0 +1,174 @@
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 1\azure}
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 2\azure}
+        & \lightcell[1.4]{1\bronze 2\silver 2\golden 3\azure}\\
+    \darkcell[1.4]{2 players} 
+	    & \lightcell[1.4]{5\bronze 5\silver 3\golden 1\azure}
+	    & \lightcell[1.4]{4\bronze 5\silver 3\golden 2\azure}
+	    & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure}
+	    & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure}\\
+    \darkcell[1.4]{3 players} 
+        & \lightcell[1.4]{8\bronze 7\silver 4\golden 2\azure}
+        & \lightcell[1.4]{6\bronze 7\silver 5\golden 3\azure}
+        & \lightcell[1.4]{2\bronze 3\silver 4\golden 3\azure \linebreak
+        	Then\linebreak 
+        	2\bronze 4\silver 3\golden 2\azure}
+        & \lightcell[1.4]{1\bronze 3\silver 5\golden 3\azure \linebreak
+        	Then\linebreak 
+        	1\bronze 4\silver 5\golden 3\azure}\\
+    \darkcell[1.4]{4 players} 
+        & \lightcell[1.4]{10\bronze 10\silver 6\golden 2\azure}
+        & \lightcell[1.4]{8\bronze 10\silver 6\golden 4\azure}
+        & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure \linebreak
+        	Then\linebreak 
+        	2\bronze 5\silver 5\golden 3\azure}
+        & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure \linebreak
+        	Then\linebreak 
+        	1\bronze 5\silver 7\golden 4\azure}\\
+	\darkcell[1.4]{5 players} 
+	  	& \lightcell[1.4]{6\bronze 6\silver 4\golden 1\azure \linebreak
+	  		Then\linebreak 
+	  		7\bronze 6\silver 3\golden 2\azure}
+	  	& \lightcell[1.4]{5\bronze 6\silver 4\golden 2\azure \linebreak
+	  		Then\linebreak 
+	  		5\bronze 6\silver 4\golden 3\azure}
+	  	& \lightcell[1.4]{3\bronze 6\silver 6\golden 4\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 6\silver 6\golden 4\azure}
+	  	& \lightcell[1.4]{1\bronze 6\silver 8\golden 6\azure \linebreak
+	  		Then\linebreak 
+	  		2\bronze 6\silver 8\golden 5\azure}\\
+	\darkcell[1.4]{6 players} 
+	  	& \lightcell[1.4]{8\bronze 8\silver 4\golden 1\azure \linebreak
+	  		Then\linebreak 
+	  		7\bronze 7\silver 5\golden 2\azure}
+	  	& \lightcell[1.4]{6\bronze 8\silver 4\golden 3\azure \linebreak
+	  		Then\linebreak 
+	  		6\bronze 7\silver 5\golden 3\azure}
+	  	& \lightcell[1.4]{3\bronze 8\silver 7\golden 5\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 7\silver 8\golden 4\azure}
+	  	& \lightcell[1.4]{3\bronze 7\silver 11\golden 6\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 8\silver 10\golden 6\azure}\\
+  \end{tabularx}
+}
+
+\vspace{3em}
+
+\center\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
+\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
+\center\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
+\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
+\center\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
+\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}
```

Comment:
Avoid using `\center` as it can cause issues
```suggestion
\begin{center}
\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}
\end{center}
```

Reply:
See my other comment about maps https://github.com/qwrtln/Homm3BG-mission-book/pull/9#discussion_r1652142140